deploy/manifests/prod/us-east-2/tenant/storetheindex/instances/inga/pvc_addata.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: addata
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 3Ti
+  storageClassName: io2
```

Comment:
Let's use `gp3` volumens for both data and ad data

Reply:
Changed